packages/react/src/FormErrorList/FormErrorList.tsx

```diff
@@ -24,6 +22,8 @@ export type FormErrorListProps = {
   errorCountLabel?: { plural: string; singular: string }
   /** The list of error messages to display. */
   errors: FormError[]
+  /** Whether the component receives focus on first render */
+  focusOnRender?: boolean
```

Comment:
I think we can name this [autofocus](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus).